lib/pbench/cli/server/reindex.py

```diff
@@ -69,38 +77,45 @@ def datasets(kwargs) -> Query[Dataset]:
     return query.filter(*filters)
 
 
-def summarize_index(dataset: Dataset, kwargs):
+def opensearch(
+    operation: str, dataset: Dataset, options: dict[str, Any], params: dict[str, str]
+) -> Optional[requests.Response]:
+    es_url, ca_bundle = options.get("_es")
+    indices = ",".join(IndexMap.indices(dataset))
+    if not indices and (detailer.detail or not options.get("reindex")):
+        click.echo(f"{dataset.name} is not indexed")
+        return None
```

Comment:
My brain's a little fuzzy right now, but I'm pretty sure that this code isn't constructed properly:  I _think_ we want to return if there are no indices, but we only want to do the print under certain circumstances.  E.g.,
```python
    if not indices:
        if detailer.detail or not options.get("reindex"):
            click.echo(f"{dataset.name} is not indexed")
        return None
```
In particular, I don't think we want to skip the `return` when `indices` is false-y just because we're reindexing and not displaying details.